src/sentry/static/sentry/app/views/alerts/details/chart.tsx

```diff
@@ -98,6 +104,107 @@ const Chart = (props: Props) => {
     })
   );
 
+  const lineSeries: LineChartSeries[] = [
+    {
+      // e.g. Events or Users
+      seriesName,
+      dataArray: chartData,
+      data: [],
+      markPoint: MarkPoint({
+        data: [
+          {
+            labelForValue: seriesName,
+            seriesName,
+            symbol: `image://${startedSymbol}`,
+            name: t('Alert Triggered'),
+            coord: startedCoordinate,
+          },
+          ...(closedTs
+            ? [
+                {
+                  labelForValue: seriesName,
+                  seriesName,
+                  symbol: `image://${closedSymbol}`,
+                  symbolSize: 24,
+                  name: t('Alert Resolved'),
+                  coord: closedCoordinate,
+                },
+              ]
+            : []),
+        ] as any, // TODO(ts): data on this type is likely incomplete (needs @types/echarts@4.6.2)
+      }),
+    },
+    warningTrigger &&
+      warningTriggerAlertThreshold && {
+        seriesName: 'Warning Alert',
```

Comment:
Needed to switch these from `name` to `seriesName` as I'm fairly certain they were all being set to `undefined` because of how the map in linechart works.